### PR TITLE
settings and camera preview categories distinction (front-end)

### PR DESCRIPTION
**Pull request by Keeran10**
This pull request will complete issue #27. 

A screenshot of the app post-commits is available in the issue comments.

### CODE REVIEW

**Review by MBenchimol — APPROVED** [review]

**Review by irinafakotakis — APPROVED** [review]
Much clearer!

**Comment by Keeran10** [comment]
Note, the build error is caused by having merged the pull request before the build process had been completed. The reason the build was in process was because I mistakenly closed the pull request without merging after a successful build and getting approval by my colleagues. Upon reopening the pr and merging, CI did not have time to complete hence the error.

**Pull request merged**
